scripts/us_census/acs5yr/subject_tables/common/data_loader.py

```diff
@@ -188,7 +191,8 @@ def __init__(self,
             # to be to run with --option=colmap. So rephrasing this message will help.
             print(
                 """To run this module in standalone mode, ensure the column map
-                is generated by running `generate_col_map` module.""")
+                is generated by running `generate_col_map` module or
+                process.py with --option=<colmap or all>.""")
```

Comment:
Can we rephrase this help message to something as follows:
"To run this module in standalone mode, ensure the column is generated first. Column map for a subject table dataset can be generated through process.py with `--option` flag set as either `colmap` or `all`. Alternatively, you can call the relevant method in `common/generate_col_map.py` module based on the input file type"

Reply:
done!